@ecomplus/storefront-components/src/js/TheProduct.js

```diff
@@ -112,6 +112,7 @@ export default {
       default: true
     },
     hasQuantitySelector: Boolean,
+    alwaysShowTimer: Boolean,
```

Comment:
Booleano é `is`, `can` ...

Reply:
E não acho que deveria adicionar uma prop pra isso, o `hasPromotionTimer` pode passar a ser `[String, Boolean]`, e aí poderia ser `false` (default), `true` ou `'always'`

Reply:
Com duas props fica uma API confusa:

```vue
<the-product
  :has-promotion-timer="false"
  :always-show-timer="true"
/>
```

Ambíguo..